internal/proxy/attribute/attribute.go

```diff
@@ -18,4 +27,13 @@ type Attribute struct {
 	Path   string        `yaml:"path" mapstructure:"path"`
 	Params []string      `yaml:"params" mapstructure:"params"`
 	Value  string        `yaml:"value" mapstructure:"value"`
+	Source string        `yaml:"source" mapstructure:"source"`
+}
+
+func ComposeAttribute(attribute string, attrs map[string]interface{}) string {
+	if strings.Contains(attribute, "{") {
+		template := fasttemplate.New(attribute, "{", "}")
```

Comment:
Should use double curly bracket for the delimiter

```suggestion
		template := fasttemplate.New(attribute, "{{", "}}")
```

Reply:
use ${} instead so yaml parser won't assume the value as map